src/parser.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Copyright 2018 ?

Reply:
Updated.  I've been rebasing this change since it was first made in 2015.